src/App.js

```diff
@@ -0,0 +1,37 @@
+import React, {useState, useEffect} from "react";
+import "./App.css";
+import Player from "./Player";
+import {db} from "./firebase";
+import 'moment-timezone';
+
+function App() {
+  const [players, setPlayers]=useState([]);
+  const currentDate=new Date().toLocaleDateString();
```

Comment:
i see `moment` as a dependency. Did we want to use that here instead of a browser `Date` object?

Reply:
Resolved